### PR TITLE
phrasing changes

### PR DESCRIPTION
**Pull request by ryan-ludwig**
Replace white/blacklist with allow/deny

### CODE REVIEW

**Comment by eddieferrer** [comment]
@ryan-ludwig there are several additional uses, if you want to bundle them all up in one PR...

File: server/util/errorLogger.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ module.exports = {
 			})
 		],
 		meta: false,
-		requestWhitelist: ['url', 'responseTime', 'level', 'message', 'headers.host', 'headers.method'],
```
Comment (eddieferrer) [inline server/util/errorLogger.js:13]: This is part of the winston API. They are working on changing these terms for the upcoming 5.0 release, but we cannot make this modification just yet....

**Review by BoulderBrains — APPROVED** [review]
👍

**Review by emuvente — APPROVED** [review]
👍

File: server/util/errorLogger.js (hunk 1 above)
Comment (mcstover) [reply server/util/errorLogger.js]: Thanks @eddieferrer for catching this!

**Pull request merged**